cmd/restic/cmd_copy.go

```diff
@@ -186,9 +189,8 @@ func copyTree(ctx context.Context, srcRepo restic.Repository, dstRepo restic.Rep
 	}, nil)
 
 	wg.Go(func() error {
-		// reused buffer
+		defer close(idChan)
 		var buf []byte
```

Comment:
That variable is now only used when loading tree blobs. Please move it to the corresponding scope.

Reply:
The buffer is reused every iteration by the LoadBlob function. I'll restore the "reused buffer" comment